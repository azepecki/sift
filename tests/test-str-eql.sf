def int main() {

	str a = "apple";
	str b = "apple";
        
	if (str_eql(a, b)) {
		"The two strings are equal" |> <out>;
	} else {
		"The two strings are not equal" |> <out>;
	}
	
	return 0;
}
