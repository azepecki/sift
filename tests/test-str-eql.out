The two strings are equal